x-pack/plugins/security_solution/public/graphql/introspection.json

```diff
@@ -9641,6 +9641,22 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
+            "name": "excludedRowRendererIds",
```

Comment:
consider not including `excludedRowRendererIds` in exported / imported timelines